CoffeeChat/Controller/Endpoints.swift

```diff
@@ -24,36 +24,70 @@ extension Endpoint {
         Endpoint.config.host = baseURL
         Endpoint.config.commonPath = "/api/v1"
     }
+    
```

Comment:
even more minor but delete the trailing whitespace in this file (and other files in PR)